packages/orchestrator-ui-components/src/hooks/useWithOrchestratorTheme.ts

```diff
@@ -0,0 +1,9 @@
+import { EuiThemeComputed } from '@elastic/eui';
+import { useOrchestratorTheme } from './useOrchestratorTheme';
+
+export function useWithOrchestratorTheme<T>(
```

Comment:
What does the generic type do here? I don't see it being passed in when calling the hook I think

Reply:
The return type of the getStyles function IS the return type of this function, therefore a generic is used. Calling this hook without specifying the type is acceptable, since the type of getStyles is inferred.